builds/fabric8-che/plugins/keycloak-plugin-server/src/main/java/com/redhat/che/keycloak/server/KeycloakAuthenticationFilter.java

```diff
@@ -9,33 +20,50 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 public class KeycloakAuthenticationFilter extends org.keycloak.adapters.servlet.KeycloakOIDCFilter {
+    
+    private static final Logger LOG = LoggerFactory.getLogger(KeycloakAuthenticationFilter.class);
 
     public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain)
             throws IOException, ServletException {
         HttpServletRequest request = (HttpServletRequest) req;
-        String auth = request.getHeader("Authorization");
-        String uri = request.getRequestURI();
-
-        if (uri.endsWith("/api/system/state")) {
-            System.out.println("Che server status endpoint should not be secured: " + uri);
-            chain.doFilter(req, res);
-        }
+        String authHeader = request.getHeader("Authorization");
+        String requestURI = request.getRequestURI();
+        String requestScheme = req.getScheme();
 
-        if (auth == null) {
-            System.out.println("No auth header for " + request.getRequestURI());
+        if (authHeader == null) {
+            LOG.debug("No auth header for " + requestURI);
```

Comment:
we should use comma instead of + ( so we compute string only if debug is enabled)